### DISCUSSION TITLE
COM/NAV/XPDR buttons displaying correct values, but pressing them results in "Connected to Stream Deck Plugin" message

### DISCUSSION THREAD

**Discussion by blave** [nguyenquyhy/Flight-Tracker-StreamDeck#185] [Q&A]
Thus far I am seeing this in the Pipistrel, TBM, and A5, so it doesn't seem to be a/c-specific.

**Comment by nguyenquyhy** [comment]
Can you provide logs in `%appdata%\Elgato\StreamDeck\Plugins\tech.flighttracker.streamdeck.sdPlugin\`? Log file names would be like `flightstreamdeck*.log`.
